examples/aggregator/backend/Cargo.toml

```diff
@@ -14,7 +14,8 @@ futures-util = "*"
 log = "*"
 prost = "*"
 structopt = "*"
-tokio = { version = "*", features = [
+# Pinned to 0.2 because of tonic: https://github.com/hyperium/tonic/blob/master/tonic/Cargo.toml
+tokio = { version = "0.2", features = [
```

Comment:
@conradgrobler there is an issue now if some libraries use tokio 0.2 and the tokio runner is 0.3 they will not work together and fail at runtime. Is there anything in cargo deny that can be done so that at least for this dep we stick to a single version?

Reply:
I think the only option is to go back to denying multiple versions and ignoring the ones that we cannot fix.